lang/cs/Org.Apache.REEF.Wake/Remote/AuxillaryStreamingFunctions.cs

```diff
@@ -0,0 +1,275 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Org.Apache.REEF.Wake.Remote
+{
+    /// <summary>
+    /// Static class containing functions to read and write some basic types from/to stream
+    /// </summary>
+    public static class AuxillaryStreamingFunctions
```

Comment:
Yes we still need this. Binary Reader/Writer do not support async functions. So unless we get away with async from Wake we cannot directly use the Binary classes.